### PR TITLE
Bratseth/filter unstable later

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/NodeMetricsDbMaintainer.java [hunk 1]
```diff
@@ -63,6 +63,7 @@ protected boolean maintain() {
     private Collection<Pair<String, MetricSnapshot>> filter(Collection<Pair<String, MetricSnapshot>> snapshots) {
```
Comment (freva) [inline node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/NodeMetricsDbMaintainer.java:63]: This is unused now?

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/QuestMetricsDb.java [hunk 2]
```diff
@@ -152,7 +154,8 @@ private void ensureExists(String tableName) {
 
         try (SqlCompiler compiler = new SqlCompiler(engine)) {
             compiler.compile("create table " + tableName +
-                             " (hostname string, at timestamp, cpu_util float, mem_total_util float, disk_util float, application_generation long)" +
+                             " (hostname string, at timestamp, cpu_util float, mem_total_util float, disk_util float," +
```
Comment (freva) [inline node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/QuestMetricsDb.java:157]: What about the existing tables?

**Review by freva — APPROVED** [review]

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/NodeMetricsDbMaintainer.java (hunk 1 above)
Comment (bratseth) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/NodeMetricsDbMaintainer.java]: Yes, removed thanks

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/QuestMetricsDb.java (hunk 2 above)
Comment (bratseth) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/QuestMetricsDb.java:157]: They will get default values, which is not what we want here (we want true not false), but the only effect is no autoscaling for a while, so I think that's okay.

Comment (freva) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/QuestMetricsDb.java:157]: So as soon as you add a row with 6th column, that is legal even though the table was created with 5 columns, and all existing rows will get a default value of the type of the 6th column in the new row?

Comment (bratseth) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/QuestMetricsDb.java:157]: Apparently - I tested it. Couldn't find any documentation on this scenario.

**Pull request merged**